etcdserver/etcdserverpb/rpc.proto

```diff
@@ -494,6 +496,13 @@ message AuthenticateResponse {
 
 message UserAddResponse {
   ResponseHeader header = 1;
+  enum Result {
```

Comment:
we do not need this actually. we can return error on error case. We can return duplicate error or exhausted error.